app/assets/javascripts/proxy_status.js

```diff
@@ -1,8 +1,12 @@
 $(document).on('ContentLoad', function() {
+  $('#proxy-tab a').on('shown.bs.tab', refreshCharts);
   showProxies();
   loadTFTP();
+  setTab();
 });
 
+$(window).on('hashchange', setTab); //needed to handle some title action buttons
```

Comment:
Because the `hashchange` event that is fired when clicking on a link to an anchor on the same page (as is the case when selecting autosign or certificates from the drop down action) is fired on the window, not the document.